src/app/services/chart.service.ts

```diff
@@ -13,8 +13,23 @@ import * as _ from 'lodash';
 @Injectable()
 export class ChartService {
 
+    private multiChartScrubberInfo = new Subject();
+    private multiChartScrubberHover = new Subject<Boolean>();
+
+    multiChartScrubberInfo$ = this.multiChartScrubberInfo.asObservable();
```

Comment:
Is this `$` postfix a common convention in the rxjs world? If it isn't, it may be better to just make the public property just be `multiChartScrubberInfo` or `multiChartScrubberObservable` and then give the private var the same name with an `_` prefix in order to be consistent with existing JS naming conventions.